src/HIE/Bios/GHCApi.hs

```diff
@@ -263,13 +264,21 @@ withCmdFlags flags body = G.gbracket setup teardown (\_ -> body)
 
 ----------------------------------------------------------------
 
+setDeferTypeErrors :: DynFlags -> DynFlags
+setDeferTypeErrors
+  = foldDFlags (flip wopt_set) [Opt_WarnTypedHoles, Opt_WarnDeferredTypeErrors, Opt_WarnDeferredOutOfScopeVariables]
+  . foldDFlags setGeneralFlag' [Opt_DeferTypedHoles, Opt_DeferTypeErrors, Opt_DeferOutOfScopeVariables]
+
+foldDFlags :: (a -> DynFlags -> DynFlags) -> [a] -> DynFlags -> DynFlags
+foldDFlags f xs x = foldr f x xs
+
 -- | Set 'DynFlags' equivalent to "-w:".
-setNoWaringFlags :: DynFlags -> DynFlags
-setNoWaringFlags df = df { warningFlags = Gap.emptyWarnFlags}
+setNoWarningFlags :: DynFlags -> DynFlags
+setNoWarningFlags df = df { warningFlags = Gap.emptyWarnFlags}
```

Comment:
Didn't we agree to delete this function?

Reply:
Its used by `Check.hs`